src/Tribe/Service_Providers/Onboarding.php

```diff
@@ -0,0 +1,126 @@
+<?php
+namespace Tribe\Service_Providers;
+
+/**
+ * Class Onboarding
+ *
+ * @since TBD
+ *
+ * Handles the registration and creation of our async process handlers.
+ */
+class Onboarding extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * Binds and sets up implementations.
+	 *
+	 * @since TBD
+	 */
+	public function register() {
+		tribe_singleton( 'onboarding', '\Tribe\Onboarding\Main' );
+
+		$this->hooks();
+	}
+
+	/**
+	 * Set up hooks for classes.
+	 *
+	 * @since TBD
+	 */
+	private function hooks() {
+		add_action( 'tribe_common_loaded', [ $this, 'register_assets' ] );
+
+		add_action( 'admin_enqueue_scripts', tribe_callback( 'onboarding', 'localize_tour' ) );
+		add_action( 'admin_enqueue_scripts', tribe_callback( 'onboarding', 'localize_hints' ) );
+	}
+
+	/**
+	 * Register assets associated with onboarding.
+	 *
+	 * @since TBD
+	 */
+	public function register_assets() {
+		$main = \Tribe__Main::instance();
+
+		tribe_asset(
+			$main,
+			'intro-js',
+			'node_modules/intro.js/intro.js',
+			[],
+			[ 'admin_enqueue_scripts' ],
+			[
+				'groups'       => 'tribe-onboarding',
+				'conditionals' => [ $this, 'should_enqueue_assets' ],
+			]
+		);
+
+		tribe_asset(
+			$main,
+			'intro-styles',
+			'node_modules/intro.js/introjs.css',
```

Comment:
we need to either commit this to our repo or preserve this during the build process, as we remove the `node_modules` folder after the build.